index.html

```diff
@@ -1483,6 +1483,21 @@
 
 </div>
 
+## Non-Retaliation {#non-retaliation}
+
+<div class="practice">
+  <span class="practicelab" id="principle-do-not-retaliate">
+    [=Parties=] must not retaliate against [=people=] who protect their [=data=] against
+    non-essential [=processing=] or exercise rights over their [=data=].
+  </span>
+</div>
+
+Whenever people have the ability to cause a [=party=] to process less of their [=data=] or to stop
+carrying out some given set of data processing that is not essential to the service, they must be
+allowed to do so without the [=party=] retaliating, for instance by artificially removing an
+unrelated feature, by decreasing the quality of the service, or by trying to cajole, badger, or
+trick the [=person=] into opting back into the [=processing=].
+
```

Comment:
Possible addition:

When two parties have a person in common as a user of both, and one party has the ability to limit some data processing of that person's data, the other party must not retaliate (such as by decreasing the quality of service, or limiting access to other data)